### ISSUE TITLE
Show only numeral metrics

### ISSUE THREAD

**Issue by s10wen** [1000ch/gulp-stylestats#22]
I noticed `-n, --number           show only numeral metrics` in Stylestats:
https://github.com/t32k/stylestats#cli-reference

Is it possible to do that via `gulp-stylestats`? Something like;
```
stylestats: {
      options: {
        type: 'csv',
        number: true
      }
    },
```

**Comment by 1000ch** [comment]
Now, `gulp-stylestats` accepts stylestats options by passing `config` property such as following...

```javascript
const stylestats = require('gulp-stylestats');
stylestats({
  type: 'csv',
  config: {...}
});
```

And, `--number` cli option is disabling following options.

```json
{
  "published": false,
  "paths": false,
  "mostIdentifierSelector": false,
  "lowestCohesionSelector": false,
  "uniqueFontFamilies": false,
  "uniqueFontSizes": false,
  "uniqueColors": false,
  "propertiesCount": false
}
```

**Comment by 1000ch** [comment]
But it's some redundancy. I'll change the way to accept `stylestats`'s option from next major version.

**Comment by s10wen** [comment]
@1000ch thanks. So is it possible to do:

```
const stylestats = require('gulp-stylestats');
stylestats({
  type: 'csv',
  config: {
    "number": true
  }
});

**Comment by 1000ch** [comment]
Ah, it not. Because `--number` seems to be an option for **CLI** only.

Maybe following will works.

```javascript
const stylestats = require('gulp-stylestats');
stylestats({
  type: 'csv',
  config: {
    published: false,
    paths: false,
    mostIdentifierSelector: false,
    lowestCohesionSelector: false,
    uniqueFontFamilies: false,
    uniqueFontSizes: false,
    uniqueColors: false,
    propertiesCount: false
  }
});
```

**Comment by s10wen** [comment]
@1000ch ah I understand now, thanks! 👍

**Comment by 1000ch** [comment]
@s10wen Just published `v2.0.0` which accepts stylestats' options directly. Sorry for confusing.

const stylestats = require('gulp-stylestats');
stylestats({
  type: 'csv',
  published: false,
  paths: false,
  mostIdentifierSelector: false,
  lowestCohesionSelector: false,
  uniqueFontFamilies: false,
  uniqueFontSizes: false,
  uniqueColors: false,
  propertiesCount: false
});

**Comment by s10wen** [comment]
@1000ch confirmed `v2.0.0` is stable and working for me with this option 🎉
